tests/test_secretsmanager/test_secretsmanager.py

```diff
@@ -181,6 +181,125 @@ def test_get_secret_version_stage_mismatch():
     ) == cm.value.response["Error"]["Message"]
 
 
+@mock_aws
+def test_batch_get_secret_value_for_secret_id_list_with_matches():
+    conn = boto3.client("secretsmanager", region_name="us-west-2")
+
+    secret_a = conn.create_secret(Name="test-secret-a", SecretString="secret")
+    secret_b = conn.create_secret(Name="test-secret-b", SecretString="secret")
+
+    secrets_batch = conn.batch_get_secret_value(
+        SecretIdList=["test-secret-a", "test-secret-b"]
+    )
+    matched = [
```

Comment:
This has the potential to hide some problems - `secrets_batch` could (somehow) contain other secrets that are filtered out in this list.

```suggestion
    assert len(secret_batch["SecretValues"]) == 2
```
This achieves the same thing, as far as I can see.